examples/advanced/train_act_pusht/train_act_pusht.sh

```diff
@@ -0,0 +1,17 @@
+python lerobot/scripts/train.py \
+    hydra.job.name=act_pusht \
+    hydra.run.dir=outputs/train/act_pusht \
+    env=aloha \
+    env.task=AlohaInsertion-v0 \
+    dataset_repo_id=lerobot/pusht \
```

Comment:
I am not sure to understand why the env is aloha

Reply:
This file shouldn't be here. Removed.